tests/baselines/reference/classExpression4.types

```diff
@@ -1,22 +1,22 @@
 === tests/cases/conformance/classes/classExpressions/classExpression4.ts ===
 let C = class {
->C : typeof (Anonymous class)
+>C : any

```

Comment:
Weirdly enough, this logic was already done in `checkFunctionAndClassExpressionBodies`:

```TypeScript
case SyntaxKind.ClassExpression:
    forEach((<ClassExpression>node).members, checkSourceElement);
    break;
```

So the question is, why wasn't this ever doing its job?